prefect_aws/workers/ecs_worker.py

```diff
@@ -450,6 +452,25 @@ class ECSVariables(BaseVariables):
             f"{ECS_DEFAULT_MEMORY} will be used unless present on the task definition."
         ),
     )
+    container_cpu: int = Field(
+        title="CPU",
+        default=None,
+        description=(
+            "The amount of CPU to provide to the Prefect container. Valid amounts are "
+            "specified in the AWS documentation. If not provided, a default value of "
+            f"{ECS_DEFAULT_CPU} will be used unless present on the task definition."
```

Comment:
I think it would be good to make the inheritance path clearer here.

Something like this:
```
            "The amount of CPU to provide to the Prefect container. Valid amounts are "
            "specified in the AWS documentation. If not provided, the value supplied to `cpu` "
            f"for the task will be used. If neither are provided, {ECS_DEFAULT_CPU} will be used "
            "unless present on the task definition."
```